bundle/manifests/prometheus-k8s_rbac.authorization.k8s.io_v1_rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: prometheus-k8s
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
```

Comment:
Not sure if possible, but these two (service account + prometheus operator namespace) should go in the csv to be overrideable, because they may change.

Reply:
I think it's not possible having the csv create resources in namespaces that it is not created in.

Reply:
Removed everything related to rbac as discussed in slack:
* PodMonitors will be deployed when the CRD is available, they'll serve as a good base in case the user wants to edit them.
* RBAC related to the PodMonitors will be deployed by the user as it is probably environment specific and we can't control it properly with the Operator.